solidity/contracts/bridge/Bridge.sol

```diff
@@ -1517,12 +1517,14 @@ contract Bridge is
     ///         (computed using Bitcoin HASH160 over the compressed ECDSA
     ///         public key) and `redeemerOutputScript` is the Bitcoin script
     ///         (P2PKH, P2WPKH, P2SH or P2WSH) that is involved in the timed
-    ///         out request. Timed out requests are stored in this mapping to
-    ///         avoid slashing the wallets multiple times for the same timeout.
+    ///         out request.
     ///         Only one method can add to this mapping:
     ///         - `notifyRedemptionTimeout` which puts the redemption key
-    ///           to this mapping basing on a timed out request stored
+    ///           to this mapping based on a timed out request stored
     ///           previously in `pendingRedemptions` mapping.
+    ///         Only one method removes entries from this mapping:
```

Comment:
Nitpicking, but choose either :
"Only one method can add entries to this mapping:"
...
"Only one method can remove entries from this mapping:"
Or:
"Only one method adds entries to this mapping:"
...
"Only one method removes entries from this mapping:"

Reply:
https://github.com/keep-network/tbtc-v2/pull/396/commits/078f6f45a59bbe62489de378232961174446bd9d